.github/workflows/python.yml

```diff
@@ -128,6 +128,46 @@ jobs:
               with:
                   language-flag: -python
 
+    test-pubsub-ubuntu-latest:
+      runs-on: ubuntu-latest
+      needs: load-engine-matrix
+      timeout-minutes: 35
+      strategy:
+          fail-fast: false
+          matrix:
+              engine: ${{ fromJson(needs.load-engine-matrix.outputs.matrix) }}
+              python:
+                - "3.8"
+                - "3.9"
+                - "3.10"
+                - "3.11"
+                - "3.12"
+
+      steps:
+          - uses: actions/checkout@v4
+            with:
+                submodules: recursive
+
+          - name: Set up Python
+            uses: actions/setup-python@v4
+            with:
+                python-version: ${{ matrix.python }}
+
+          - name: Build Python wrapper
+            uses: ./.github/workflows/build-python-wrapper
+            with:
+                os: "ubuntu"
+                target: "x86_64-unknown-linux-gnu"
+                github-token: ${{ secrets.GITHUB_TOKEN }}
+                engine-version: ${{ matrix.engine.version }}
+
+          - name: Test pubsub with pytest
+            working-directory: ./python
+            run: |
+                source .env/bin/activate
+                cd python/tests/
+                pytest -c ../../pytest_pubsub.ini --asyncio-mode=auto test_pubsub.py::TestPubSub
```

Comment:
You can remove the pytest_pubsub.ini file and just do here:
```
pytest --asyncio-mode=auto -k test_pubsub
```
without any changes to the pytest.ini file

Reply:
right, done